edge-functions/feature-flag-apple-store/package.json

```
@@ -4,24 +4,24 @@
   "repository": "https://github.com/vercel/examples.git",
   "license": "MIT",
   "scripts": {
-    "dev": "next dev",
     "build": "next build",
-    "start": "next start",
-    "lint": "next lint"
+    "dev": "next dev",
+    "lint": "next lint",
+    "start": "next start"
   },
   "dependencies": {
-    "next": "^12.1.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "next": "12.1.7-canary.45",
```

Lets use `canary` here as it's constantly getting updates, until we can move to `latest`:
```suggestion
    "next": "canary",
```

---

https://github.com/vercel/examples/pull/318#discussion_r905162311 Check this thread